src/GitHub.VisualStudio/Settings/PackageSettingsGen.cs

```diff
@@ -0,0 +1,34 @@
+﻿// This is an automatically generated file, based on settings.json and PackageSettingsGen.tt
```

Comment:
Do we even need to commit these? Will the CI and local build generate them as part of the build?